mesos/src/main/java/brooklyn/entity/mesos/MesosClusterImpl.java

```diff
@@ -151,6 +153,17 @@ public String apply(@Nullable Entity input) {
         sensors().set(MESOS_FRAMEWORKS, frameworks);
         sensors().set(MESOS_TASKS, tasks);
         sensors().set(MESOS_APPLICATIONS, applications);
+        
+        // Override the health-check: just interested in the slaves, frameworks and sdn (rather than 
+        // the groups that show the tasks or apps).
+        Entity sdn = sensors().get(SDN_PROVIDER);
+        enrichers().add(EnricherSpec.create(ComputeServiceIndicatorsFromChildrenAndMembers.class)
+                .uniqueTag(ComputeServiceIndicatorsFromChildrenAndMembers.DEFAULT_UNIQUE_TAG)
+                .configure(ComputeServiceIndicatorsFromChildrenAndMembers.FROM_CHILDREN, true)
+                .configure(ComputeServiceIndicatorsFromChildrenAndMembers.ENTITY_FILTER, Predicates.or(ImmutableList.of(
+                        Predicates.<Entity>equalTo(slaves), 
+                        Predicates.<Entity>equalTo(frameworks), 
+                        (sdn == null ? Predicates.<Entity>alwaysFalse() : Predicates.equalTo(sdn))))));
```

Comment:
Probably didn't need this null check, since otherwise you have `Predicates.equalTo(null)` and that will never be true...